pytorch_toolkit/image_classification/model_templates/custom-classification/README.md

```diff
@@ -1,15 +1,61 @@
-
-## Datasets
-
-The model was initially trained on [ImageNet](http://image-net.org/challenges/LSVRC/2012/), but can be trained from scratch or fine tuned to classify arbitrary images.
-
+## Description of the models
+### Performance results pivot table
+
+
+| Model Name | Complexity (GFLOPs) | Size (Mp) | Latency (ms) | FPS | AVG mAP (%) | AVG Top-1 (%) | AVG Top-5 (%) | Links |
+| ---        | ---        | ---        | ---        | ---        | ---        | ---        | ---        | ---        |
+| EfficientNet b0 | 0.76 | 4.14 | 9.7 | 447.51 | 92.267 | 90.156 | 98.283 | [model template](./efficient_b0/template.yaml) |
+| MobilenetV3 large x1.0 | 0.44 | 4.33 | 5.58 | 792.22 | 92.04	| 89.019 | 97.807 | [model template](./mobilenet_v3_large_1/template.yaml) |
+| MobilenetV3 large x0.75 | 0.308 | 2.84 | 4.48 | 985.4 | 87.59 |	87.59	| 97.17 | [model template](./mobilenet_v3_large_075/template.yaml) |
+| MobilenetV3 small x1.0 | 0.112 | 1.56 | 2.79 | 1599.39 | 82.84 | 84.47 | 95.88 | [model template](./mobilenet_v3_small/template.yaml) |
+
+All of the above metrics were obtained on eleven different datasets, on which an extensive number of researches has been made.
+
+Performance mesured on CPU: Intel(R) Core(TM) i9-9820X CPU @ 3.30GHz,  OpenVINO 2021.2​, batch size=1, input resolution is 224 × 224, inference precision is FP32
+### Datasets and information about metrics
+
+The following datasets were used for experiments:
+* [Describable Textures (DTD)](https://www.robots.ox.ac.uk/~vgg/data/dtd/)<sup>1</sup>
+* [Caltech 101](http://www.vision.caltech.edu/Image_Datasets/Caltech101/)<sup>1</sup>
+* [Oxford 102 Flowers](http://www.robots.ox.ac.uk/~vgg/data/flowers/102/)
+* [Oxford-IIIT Pets](https://www.robots.ox.ac.uk/~vgg/data/pets/)
+* [CIFAR100](https://www.cs.toronto.edu/~kriz/cifar.html)
+* [SVHN (w/o additional data)](http://ufldl.stanford.edu/housenumbers/)
+* [Fashion MNIST](https://github.com/zalandoresearch/fashion-mnist)
+* [FOOD101](https://www.kaggle.com/dansbecker/food-101)<sup>1</sup>
+* [SUN397](https://vision.princeton.edu/projects/2010/SUN/)<sup>1</sup>
+* [Birdsnap](http://thomasberg.org/)<sup>1</sup>
+* [Cars Dataset](https://ai.stanford.edu/~jkrause/cars/car_dataset.html)
+
+<sup>1</sup> these datasets have custom splits (random stratified split: 80% - train, 20% - val) and cannot be compared straightforward with other research results
+
+For additional information about performance on each dataset in comparison with baseline, you can refer to this [spreadsheet](https://docs.google.com/spreadsheets/d/1CV3be-VydEHvWS6GMPduBQBjl46uLq80_GtkeUhsuVg/edit#gid=0).
+### Training recipe
+
+Baselines:
+* softmax loss
+* most appropriate average learning rate for all datasets (0.016)
+* cosine scheduler
+* basic augmentations
+* SGD with momentum optimizer
+
+MobilenetV3:
+* Mutual learning approach
+* Softmax loss for the main model, AM_softmax for the auxiliary model
+* Learning rate found by LR Finder
```

Comment:
I think we should provide a link to the torch-lr-finder. It's hard to get what LR Finder actually is and how it works.

Reply:
But in this case, the learning rate found by the brute force method, not by the fast.ai approach
Is it a good reason to add a paragraph describing the learning rate finder and how to use it? I think it is worth it

Reply:
Seems like yes, since the learning rate parameter of the template now is not that it's supposed to be

Reply:
Added description for LR Finder